service/src/main/java/com/oms/service/ModifyFulfillmentService.java

```diff
@@ -70,7 +70,7 @@ public OrderLine fetchOrderLineFromItemId(String lineItemId , SalesOrder salesOr
     }
 
     public EmailRequestDto buildEmailRequest(SalesOrder salesOrder) {
-        return new EmailRequestDto("1234","Modify fulfillment","Your line item have been successfully modified for fulfillment","Modify to shipping from store pickup");
+        return new EmailRequestDto(salesOrder.getCustomerOrderId(), "Modify fulfillment", "Your line item has been successfully modified for fulfillment", "Modify to shipping from store pickup");
```

Comment:
I don't think this matches the parameters of the EmailRequestDto constructor EmailRequestDto(String to, String subject, String body, String from)